src/App.js

```diff
@@ -37,7 +37,9 @@ const tweetsArray = [
 function App() {
   return (
     <div className="App">
-      <Tweet />
+      <Tweet tweet={tweetsArray[0]} />
```

Comment:
Here, you could've mapped the tweetsArray, like: 

`{tweetsArray.map((el) =>{
      return <Tweet tweet={el}/>  
    })}`